src/ConnectionWrapper.php

```diff
@@ -27,7 +27,7 @@
  *
  * This class is not tied with the Drupal Queue API.
  */
-class Connection {
+class ConnectionWrapper {
```

Comment:
Any reason to rename that class ? There is no collision in the namespace, so what is the point ?